### DISCUSSION TITLE
Discipline names, genre names, and other labels

### DISCUSSION THREAD

**Discussion by ybronner** [PanditProject/Website#40] [Help]
I am charged with the task of expanding our label list: whenever you try to enter a label (genre, discipline, language, and social identifier) and can't find it in the menu, you enter it under "other," and I later clean this up: I create a new label, and in the entity in question, I remove it from the "other" field into the ordinary place. In connection with this, I have a few questions on which I need input from others:
1. Should "Uṇādi" be a discipline?
2. Should "Paribhāṣā" be a discipline?
3. Should "Mugdhabodha" be a discipline? 
4. Should "Dhāṭupāṭha" be a discipline?
5. Should "Prakriyā" be a discipline?
6. Should "Stuti" be a genre in addition to the already existing "stava"?
7. Should "Lekhapaddhati" be a discipline?
We can start with these, I might add more as this discussion develops. 
Thanks for any input, Yigal

**Comment by chakrabortydeepro** [comment]
I think,
1. Uṇādi should be a discipline. It definitely comes under Vyākaraṇa. But we have some disciplines like Kātantra and Pāṇinīya which also come under Vyākaraṇa. It would be useful for researchers who are particularly interested in Uṇādi works, both sūtras and commentaries. Most of the Vyākaraṇa schools have their own Uṇādi texts and there are quite a lot such works. 
2. For the same reason Paribhāṣā, Prakriyā and Dhātupāṭha should be considered disciplines. 
3. There are several Mugdhabodha works. If Kātantra can be a discipline, why not Mugdhabodha. 
6. Stuti and Stava are same. 
7. There aren't many Lekhapaddhati texts. But Sahib Ram and his son Damodar Shastri's such works cannot probably be kept under any other discipline. They themselves are calling these works 'Lekhapaddhati'.

**Comment by petersonjonath** [comment]
I generally agree with Deepro’s grouping, but I'm wondering how we are distinguishing between genre and discipline. A quick search on PANDiT doesn’t make this clear. Forgive me if it’s already spelled out there.



On Oct 13, 2021, at 11:51 PM, Deepro Chakraborty ***@***.******@***.***>> wrote:



I think,

  1.  Uṇādi should be a discipline. It definitely comes under Vyākaraṇa. But we have some disciplines like Kātantra and Pāṇinīya which also come under Vyākaraṇa. It would be useful for researchers who are particularly interested in Uṇādi works, both sūtras and commentaries. Most of the Vyākaraṇa schools have their own Uṇādi texts and there are quite a lot such works.
  2.  For the same reason Paribhāṣā, Prakriyā and Dhātupāṭha should be considered disciplines.
  3.  There are several Mugdhabodha works. If Kātantra can be a discipline, why not Mugdhabodha.
  4.  Stuti and Stava are same.
  5.  There aren't many Lekhapaddhati texts. But Sahib Ram and his son Damodar Shastri's such works cannot probably be kept under any other discipline. They themselves are calling these works 'Lekhapaddhati'.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/PanditProject/Website/discussions/40#discussioncomment-1474977>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUDNA35CCZL3VOMEJMUFIM3UGZ4XFANCNFSM5F5SGG2A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by ybronner** [reply]
We actually do have a brief definition of genre and discipline for Pandit [here](https://www.panditproject.org/info/types), but I admit it could be improved. In general, discipline is a field or subfield, like Vyākaraṇa or Pāṇinīya, respectively, which could be written in any style of writing: sūtra text, commentary, etc. Genre has to do with the style and generic attributes of a work, which in some cases can cross discipline lines (vyākhyā in medicine or in grammar, for instance). In some ways, it is important to have as many as possible accurate labels for a work, which will help search and create 2nd order data about it (graphs, charts, etc.), and this is perhaps more important than whether the label belongs in this or that category. This does not mean though that we should not try to be exact. 
That said, there are murky areas between and within the two. For example of the second, I guess we don't know for sure whether or not the labeling of a work as a stava vs. a stuti is meaningful. Obviously, many works come with alternative names (the same work can be called both X-stuti and X-stava), but maybe it is sometimes not meaningless and worth recording when they come only with one. I don't know. We created separate labels for vyākhyā, bhāṣyā, vṛtti, ṭīkā, etc. as these seemed meaningful to record. Stuti-stava may be different.
Happy for more input on these questions.
